src/applications/vaos/reducers/newAppointment.js

```diff
@@ -68,6 +110,14 @@ export default function formReducer(state = initialState, action) {
       };
     }
     case FORM_DATA_UPDATED: {
+      let newPages = state.pages;
+      if (
+        action.data.typeOfCareId !== state.data.typeOfCareId &&
```

Comment:
If someone goes back through the form and changes their type of care, we'll have to pull new facilities, so this clears the page schema for now. When they get back to the facility page, it'll recreate it based on the data we have and fetch whatever is needed.